.github/workflows/test.yml

```diff
@@ -15,15 +15,16 @@ jobs:
 
     services:
       textile:
-        image: textile/textile:1.0.3
+        image: textile/textile:latest
```

Comment:
living dangerously 😉

Reply:
yep, wont affect @textile/textile which is locked on the last release... forever.